packages/pantheon/hooks/jira-auth-hook.py

```
@@ -0,0 +1,487 @@
+            return Path(candidate).expanduser()
+    return Path(DEFAULT_TEMP_ROOT)
+
+
+def lookup_token(current_profile):
+    custom = os.environ.get(TOKEN_CMD_ENV)
+    token_cmd = custom or DEFAULT_TOKEN_CMD.format(
+        profile_arg=shlex.quote(f"jira:{current_profile}")
+    )
+    log(f"token lookup via {'HARNX_JIRA_TOKEN_CMD' if custom else 'secret-tool'}: {token_cmd!r}")
+    try:
+        completed = subprocess.run(
```

Ruff (0.15.21)

[error] `subprocess` call with `shell=True` identified, security issue

(S602)